durabletask/worker.py

```diff
@@ -88,15 +90,25 @@ def __init__(self, *,
                  metadata: Optional[list[tuple[str, str]]] = None,
                  log_handler=None,
                  log_formatter: Optional[logging.Formatter] = None,
-                 secure_channel: bool = False):
+                 secure_channel: bool = False,
+                 interceptors: Optional[list[grpc.ServerInterceptor]] = None):  # Add interceptors
```

Comment:
This might be just my ignorance on gRPC + Python, but `grpc.ServerInterceptor` sounds like something you'd put on the server, whereas the worker is more like a client. Is this the right type for us to be using or in Python is a `ServerInterceptor` also used for clients?